chrome/locale/de/quicktext.dtd

```diff
@@ -75,16 +75,16 @@
 
 <!ENTITY quicktext.other.label "Andere">
 <!ENTITY quicktext.clipboard.label "Zwischenablage">
-<!ENTITY quicktext.header.label "Header (To, Cc, Bcc) hinzufügen">
-<!ENTITY quicktext.selection.label "Selektion">
-<!ENTITY quicktext.cursor.label "Cursor positionieren">
+<!ENTITY quicktext.header.label "Kopfzeile (To, Cc, Bcc) hinzufügen">
+<!ENTITY quicktext.selection.label "Auswahl">
+<!ENTITY quicktext.cursor.label "Mauszeiger positionieren">
```

Comment:
Mauszeiger ist hier falsch. Entweder Textcursor, Cursor oder Caret:
https://de.wikipedia.org/wiki/Cursor

Ich bin für Cursor.

Reply:
We should stick to "Cursor" (or "Textcursor").

https://de.wikipedia.org/wiki/Pfeiltaste

Reply:
https://de.wikipedia.org/wiki/Mauszeiger
https://wiki.gnome.org/de/StandardUebersetzungen

Wie wäre es mit »Eingabemarke«?